script/lib/task.js

```diff
@@ -0,0 +1,138 @@
+/**
```

Comment:
Can't we make this an npm package? This allows more flexibility for updating it.